### PR TITLE
fix(events-explorer): events explorer improvements

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

- The events explorer table is slow in its default view for our team (10sec for 24h of events), and OOMs for some of our largest users due to the persons table join.
- Without the join results always come back in under a second.

## Changes

- tldr: makes the events explorer table usable
- Uses a separate query to fetch persons, just like we did in the old events table.
- This applies just to selecting a special column "person" in "EventsQuery" that powers the events table, not to HogQL itself.
- Person property filters would still rely on a join like they did before
- Also flyby updates some example queries, limiting their timeranges.
- Also flyby changes the events table "live update" interval from 5 sec to 30sec.

## How did you test this code?

Added a test and clicked around locally.

### CODE REVIEW

File: posthog/models/event/events_query.py [hunk 1]
```diff
@@ -154,33 +150,40 @@ def run_events_query(
...
+        for person in persons:
+            if person:
+                for distinct_id in person.distinct_ids:
+                    distinct_to_person[distinct_id] = person
+
+        # Loop over all columns in case there is more than one "person" column
+        for column_index, column in enumerate(select_input_raw):
+            if column != "person":
+                continue
+            for index, result in enumerate(query_result.results):
```
Comment (pauldambra) [inline posthog/models/event/events_query.py:171]: I'm not super clear on what all is going on in these things yet 🙈 

This is an in-memory loop not an N+1 query?

**Review by pauldambra — COMMENTED** [review]
Do I need to do some setup to make this work?

where-they-be

File: posthog/models/event/events_query.py (hunk 1 above)
Comment (mariusandra) [reply posthog/models/event/events_query.py:171]: These are all in memory loops.

To get it to work, just select more than "last 24h"

Comment (pauldambra) [reply posthog/models/event/events_query.py:171]: > To get it to work, just select more than "last 24h"

🙈 🤣 

Then I get

Screenshot 2023-03-09 at 10 28 49

Comment (pauldambra) [reply posthog/models/event/events_query.py:171]: `return !!personRecord.distinct_ids.length ? (`

when

```
{
    "uuid": "01869ed9-85a3-0000-f0dc-c6dbf095aef6",
    "created_at": "2023-03-01T20:24:59.852Z",
    "properties": {
        "name": null,
        "email": "paul@posthog.com"
    },
    "distinct_id": "a0JNkovhz54zqTBN2pBKf8jqA488qudRnugUzVshN2E"
}
```

Comment (pauldambra) [reply posthog/models/event/events_query.py:171]: Dumping in 

```
else if (key === 'person' && isEventsQuery(query.source)) {
        const personRecord = value as PersonType
        let distinctId = undefined
        if (personRecord.distinct_ids && personRecord.distinct_ids.length > 0) {
            distinctId = personRecord.distinct_ids[0]
        } else if (personRecord.distinct_id) {
            distinctId = personRecord.id
        }
        return distinctId ? (
            <Link to={urls.person(distinctId)}>
                <PersonHeader noLink withIcon person={personRecord} />
            </Link>
        ) : (
            <PersonHeader noLink withIcon person={value} />
        )
    }
```

and the table displays (although the type system is unhappy)

**Review by pauldambra — APPROVED** [review]
other than the `distinct_id` vs `distinct_ids` kerfuffle :shipit:

**Pull request merged**